docs/source/CONTRIBUTING.md

```diff
@@ -10,14 +10,15 @@ expected of anyone who participates in the project by reading these Contribution
 Guidelines.
 
 **Please Note:** GitHub is for bug reports and contributions primarily - if you
-have a support question head over to [GoDaddy's Open Source Software Slack][slack].
+have a support question head over to [GoDaddy's Open Source Software Slack][slack] or the
+[Tartufo Mailing list].
 
 ## Answering Questions
 
 One of the most important and immediate ways you can support this project is to
-answer questions on [Slack][slack] or [Github][issues]. Whether you're helping a
-newcomer understand a feature or troubleshooting an edge case with a seasoned
-developer, your knowledge and experience with Python or security can go a long
+answer questions on [Slack][slack], [Github][issues] or the [Tartufo Mailing list].
```

Comment:
```suggestion
answer questions on [Slack][slack], [Github][issues], or the [Tartufo Mailing list].
```
(I wasn't going to nitpick on this, but since I had another change, I just have to offer up the Oxford comma)